app/.gitignore

```diff
@@ -1,2 +1,3 @@
 /build
-google-services.json
\ No newline at end of file
+google-services.json
+*.ws.kts
```

Comment:
What is that?

Reply:
Files ending in `.ws.kts` are Kotlin worksheets. So here it states that Git should ignore worksheets.